### DISCUSSION TITLE
Question regarding channel fees

### DISCUSSION THREAD

**Discussion by jaxkeiser** [lightningnetwork/lnd#5968] [Q&A]
How can I find out what I should set `--base_fee_msat` and `--fee_rate` to for an individual channel in order to be competitive?

**Comment by xraid** [comment]
i think it is hard to have a definite answer, it has to be in relation to Your node´s connected peers, general recommendation would be "trial" also considering network is alive and changing regardless while Your change and changes are not reflected at same minute, so again depends on Your peers connectivity and fees ... try map your peers fees and place Yourself at desired expected outcome, this will only be effective depending peers balances etc.
